### PR TITLE
Add auto plugin registration to FlutterFragmentActivity as well

### PR DESCRIPTION
**Pull request by xster**
Fixes https://github.com/flutter/flutter/issues/54386

### CODE REVIEW

File: shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java [hunk 1]
```diff
@@ -710,4 +716,34 @@ protected RenderMode getRenderMode() {
...
+   * GeneratedPluginRegistrant must be generated per app, because each app uses different sets of
+   * plugins. Therefore, the Android embedding cannot place a compile-time dependency on this
+   * generated class. This method uses reflection to attempt to locate the generated file and then
+   * use it at runtime.
+   *
+   * <p>This method fizzles if the GeneratedPluginRegistrant cannot be found or invoked. This
+   * situation should never occur, but if any eventuality comes up that prevents an app from using
+   * this behavior, that app can still write code that explicitly registers plugins.
+   */
+  private static void registerPlugins(@NonNull FlutterEngine flutterEngine) {
```
Comment (jason-simmons) [inline shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java:734]: Looks like this is duplicated from `FlutterActivity`.

Can the `FlutterActivity` method be made package protected and invoked here?  Or maybe this should be a utility method somewhere in the `io.flutter.embedding.engine.plugins` package?

Comment (xster) [reply shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java]: oh ya, good idea. Done.

**Comment by xster** [comment]
Rerun completed on luci. Not updating on github for some reason. Merging
Screen Shot 2020-08-31 at 1 42 57 PM

**Comment by ened** [comment]
@xster should this be cherry-picked onto stable? The `local_auth` plugin depends on this.

**Comment by xster** [comment]
This is already in beta and should be in stable within a week.

**Pull request merged**